✨✏️ Removed .env from .gitignore

ShivamB25 opened the pull request.

ShivamB25 committed:
feat: Add Docker configurations for the project

Added Docker Compose and Dockerfile to the project to enable containerization. This allows for easier deployment and management of the application.

ShivamB25 committed:
✨✏️ Removed .env from .gitignore

The `.env` file was accidentally excluded from the `.gitignore` file. This can cause sensitive information to be committed to the repository. To fix this, we removed `.env` from the `.gitignore` file.

✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅

ShivamB25 reviewed (commented):
bot.js can be probably improved.

VoloBuilds commented:
Hey @ShivamB25!! Thanks for the PR! Just wanted you to know I saw it and I will review later today :)

VoloBuilds commented on .env.sample:
This is a good idea, thanks

VoloBuilds commented on .gitignore:
Could you reduce the .gitignore back down to only reasonable stuff for this repo? All of those extra ignores add bloat to the repo and may confuse users as to why they are included. Let's just have node_modules and .env in the gitignore (which I think was your main goal anyway)

VoloBuilds commented on Dockerfile:
Great! Thank you for this

VoloBuilds commented on README.md:
```diff
@@ -38,6 +38,7 @@ https://youtu.be/5TU_wOC0dmw
 
 1. **Set Up Environment Variables**:
    Create a `.env` file in the root of your project with the following variables:
+   mv .env.sample .env 
```
👍

VoloBuilds commented on bot.js:
Let's keep the original 4-spaces style; please revert the changes to bot.js. Feel free to use 2-space in your fork if you wish!

VoloBuilds reviewed (changes requested):
Thank you for the PR! Really appreciate the Docker additions and the .env change is good too. Left comments; let me know if you can make the changes so I can merge!

ShivamB25 committed:
🔧 Refactor .gitignore and bot.js

- Refactor `.gitignore` to remove unused cached files and directories.
- Refactor `bot.js` to improve code structure, readability, and maintainability.

ShivamB25 reviewed (commented):
ok

ShivamB25 reviewed (commented):
test

VoloBuilds reviewed (approved):
Fantastic, thanks @ShivamB25! Merging
